### PR TITLE
DOCS-7142 made edits

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/manage-versions-instances-concept.adoc [hunk 1]
```diff
@@ -3,25 +3,24 @@ ifndef::env-site,env-github[]
...
+== Managing API Instances in API Manager
 
-In API Manager, you may also create, import, and deprecate API instances.
+You can create, import, and deprecate API instances using API Manager. You can use instances of an API in multiple environments. You can also have multiple instances of an API in the same environment:
 
-On the API administration dashboard, you expand the API version name to display the instances if there are any:
+//image::jsonplaceholder2.png[]
 
-image::jsonplaceholder2.png[]
+You can create one instance of an API that serves as a proxy and create another to directly manage as a basic endpoint. You can apply policies to the API based on how you have configured the endpint. For example, you can apply caching policies to the proxy, and throttling policies and security policies to the basic endpoint.
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-versions-instances-concept.adoc]: version?
```suggestion
You can create one instance of an API version that serves as a proxy and create another to manage directly as an endpoint. You can apply policies to the API based on how you configure that endpoint. For example, you can apply caching policies to the proxy, and throttling policies and security policies to the endpoint. 
```
The example is not an example of the sentence before it, which tells you that you can apply policies to the API based on how you configure the endpoint but does not mention applying policies to the API through the proxy.
It is unclear here which is the dominant/manager instance: the endpoint or the proxy.

File: modules/ROOT/pages/manage-versions-instances-concept.adoc [hunk 2]
```diff
@@ -3,25 +3,24 @@ ifndef::env-site,env-github[]
...
 
-On the API administration dashboard, you expand the API version name to display the instances if there are any:
+//image::jsonplaceholder2.png[]
 
-image::jsonplaceholder2.png[]
+You can create one instance of an API that serves as a proxy and create another to directly manage as a basic endpoint. You can apply policies to the API based on how you have configured the endpint. For example, you can apply caching policies to the proxy, and throttling policies and security policies to the basic endpoint. 
 
-You can use instances of an API in multiple environments, and you can have multiple instances in the same environment. You can create one instance of an API that serves as a proxy. You can create another to manage directly as a basic endpoint. You can apply caching policies to the proxy, for example, and throttling policies and security policies to the basic endpoint. For more information about API instance management, see "About Basic API Management Tasks" link below.
+Additionally, you can promote an API to another environment.
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-versions-instances-concept.adoc:20]: Is this called "promoting"? Does that mean that you can share the API, create a new instance, or move the API entirely? Can you then change the shared/moved instance independently of the original?

File: modules/ROOT/pages/manage-versions-instances-concept.adoc [hunk 3]
```diff
@@ -3,25 +3,24 @@ ifndef::env-site,env-github[]
...
+//(Promoting an API requires Manage APIs Configuration permission to be set for a user.)
 
-In addition to API instance management in API Manager, you can perform other API management tasks. You can promote an API to another environment and configure an API endpoint, for example. (Promoting an API requires Manage APIs Configuration permission to be set for a user.)
+== Managing API Versions in Exchange
 
-== API Version Management in Exchange
-
-In Exchange you may add, delete, and deprecate API versions, which are called Exchange assets. An API version in Exchange can have its own RAML file, API Portal, and URL. For more information about API version management in Exchange, see "Anypoint Exchange" documentation.
+You can add, delete, and deprecate API versions, also known as Exchange assets, in Anypoint Exchange (Exchange). An API version in Exchange can have its own RAML file, API Portal, and URL.
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-versions-instances-concept.adoc]:
```suggestion
You can add, delete, and deprecate API versions, also known as _Exchange assets_, in Anypoint Exchange (Exchange). An API version in Exchange can have its own RAML file, API portal, and URL. 
```
This text reads as if it were introductory information about versioning. Should it be moved up in the text chronology?

**Review by kbostrom — APPROVED** [review]
Done!

File: modules/ROOT/pages/manage-versions-instances-concept.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/manage-versions-instances-concept.adoc]: Fixed! Thanks!

File: modules/ROOT/pages/manage-versions-instances-concept.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/manage-versions-instances-concept.adoc:20]: No promote is OK. And promote means that you can move an API from one environment to another, for example, from testing to production.

File: modules/ROOT/pages/manage-versions-instances-concept.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/manage-versions-instances-concept.adoc]: No it is Ok where it is because it is about what you can do with an API version.

**Pull request merged**